package.json

```diff
@@ -5,7 +5,7 @@
   "author": "Sashko Stubailo",
   "main": "lib/index.js",
   "scripts": {
-    "test": "tav --ci && mocha test/index.js",
+    "test": "tav",
```

Comment:
Tests have been failing in CI for a while, because in CI tests rung against all versions of `graphql` that satisfy the semver ranges defined in `.tav.yml`: https://github.com/apollographql/eslint-plugin-graphql/blob/ed2e619f884be8531f342f08c7bf3e845d47fc74/.tav.yml#L3

Amongst those was `v14.3.1`, which shows a slightly different error message for the `ProvidedRequiredArguments` validator. You can see the error here: https://travis-ci.org/apollographql/eslint-plugin-graphql/jobs/548949602

If you look at recent PRs, all of them are failing with the same error. What made this confusing to me was that [`tav`](https://github.com/watson/test-all-versions) was only being run on CI, and not locally. So locally tests were passing. For that reason, I thought it would be best if `tav` also runs locally now –– hence why the `--ci` flag is removed here. If that's not okay, I can revert this part of the PR.

Anyways, the fix for the actual error can be seen below in `test/validationRules/built-in.js`